data/transactions/verify/txn.go

```diff
@@ -361,7 +374,7 @@ func logicSigSanityCheckBatchPrep(gi int, groupCtx *GroupContext, batchVerifier
 	if version > groupCtx.consensusParams.LogicSigVersion {
 		return errors.New("LogicSig.Logic version too new")
 	}
-	if uint64(lsig.Len()) > groupCtx.consensusParams.LogicSigMaxSize {
+	if !groupCtx.consensusParams.EnableLogicSigSizePooling && uint64(lsig.Len()) > groupCtx.consensusParams.LogicSigMaxSize {
```

Comment:
Since you're here, I'd prefer this error message change to `"LogicSig too long"` since `Len()` is (correctly) reporting the sum of `Logic` and `Args`

Reply:
you are correct, changed